docs/user/reference/openapi.rst

```diff
@@ -0,0 +1,5 @@
+OpenAPI Specification
```

Comment:
You can actually get sphinx to render the spec in a swagger-like way with a plugin, see example in scanspec: https://github.com/dls-controls/scanspec/blob/master/docs/reference/rest_api.rst

Also, I think we should rethink calling the docs "openapi.rst" and "asyncapi.rst", as that only defines the type, not the purpose, of the API they are documenting.

Reply:
Yes, I agree. I do think however that renaming the docs is maybe a separate issue that can tie into #181?

Reply:
Gonna have a look at that plugin though.